### DISCUSSION TITLE
New Set of Code Behind templates available in the Market Place

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#4159] [Show and tell]
![image](https://github.com/oqtane/oqtane.framework/assets/9447612/fd0690e6-7e52-4e8e-9407-bbd2478ffb88)

Template package has had Radzen added.  This is currently only for the Module template, I will be adding the Radzen Theme template shortly.

**Comment by sbwalker** [comment]
This is excellent @leigh-pointer !! This is exactly why the templates were created in an extensible manner - to allow people to use custom templates

**Comment by leigh-pointer** [comment]
Just started Synfusion template. Another one for the pot
________________________________

**Comment by leigh-pointer** [comment]
Syncfussion is complete but will release a new set when Telerik and DevExpress are incorportated, unless someone wants Syncfussion now.

**Comment by rchibnik** [comment]
My rsdzen component via this template works great. Thanks for this excellent work.

Unfortunately, Icons are not showing up. probaly missing a style include / copy. 

Any suggestions?

Thanks

**Reply by leigh-pointer** [reply]
Hi @rchibnik 
I have search the Radzen site and found 

- https://forum.radzen.com/t/icons-not-working-in-layout/16975
- https://forum.radzen.com/t/no-icons-in-radzengrid/4144
- https://forum.radzen.com/t/material-icons-missing/12926

none of which worked but,
in the `public override List<Resource> Resources => new List<Resource>()`

add this line
`new Resource { ResourceType = ResourceType.Stylesheet, Url = "https://fonts.googleapis.com/icon?family=Material+Icons"},`

and you should be ok to go.
Happy coding

**Comment by markdav-is** [comment]
@leigh-pointer did you happen to release Telerik templates.  I'm getting some request at my new job to use Telerik and would love to use your templates.

**Reply by leigh-pointer** [reply]
@markdav-is i did start but their setup is a nightmare! I haven't checked since net9 so might be worth looking again!

**Reply by markdav-is** [reply]
I'm trying to avoid it if I can.
